db/template_test.go

```diff
@@ -0,0 +1,131 @@
+// Package db - template tests
+// The following tests validate database functionality, the instance payload.
+// A postgres instance with the tink database schema is required in order to run
+// the following tests.
+package db
+
+import (
+	"database/sql"
+	"fmt"
+	"sync"
+	"testing"
+
+	"github.com/google/uuid"
+	_ "github.com/lib/pq"
+)
+
+// TestCreateTemplate : validates the creation of a new workflow template.
+func TestCreateTemplate(t *testing.T) {
```

Comment:
ditto like TestInsertIntoDB can we name it `setupTemplateTests` so all the other tests don't look weird running other tests :D

Reply:
I have these tests because they test the core functionality of each package. I then call them from all the other tests that require them. This way, if we just wanted to test the Create<Object> function, we could just test that. If you don't think that's necessary, I will change them to setup<Func>.

Reply:
Changed to setup<Func>